app/views/families/_form.html.erb

```diff
@@ -35,7 +35,7 @@
     </li>
     <li>
       <%= f.label Family.human_attribute_name(:active) %>
-      <%= f.input_field :active, as: :radio_buttons, collection: ["Y", "N"] %>
+      <%= f.input_field :active, as: :radio_buttons, collection: ["true", "false"] %>
```

Comment:
I think that we can remove the explicit collection here

Reply:
Done in sha `8b313ef`